From: "Matthew Wilcox (Oracle)" <willy@infradead.org>
Subject: mm: delete thp_nr_pages()
Date: Wed, 2 Apr 2025 22:06:10 +0100

All callers now use folio_nr_pages().  Delete this wrapper.

Link: https://lkml.kernel.org/r/20250402210612.2444135-9-willy@infradead.org
Signed-off-by: Matthew Wilcox (Oracle) <willy@infradead.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/mm.h |    9 ---------
 1 file changed, 9 deletions(-)

--- a/include/linux/mm.h~mm-delete-thp_nr_pages
+++ a/include/linux/mm.h
@@ -2168,15 +2168,6 @@ static inline long compound_nr(struct pa
 }
 
 /**
- * thp_nr_pages - The number of regular pages in this huge page.
- * @page: The head page of a huge page.
- */
-static inline long thp_nr_pages(struct page *page)
-{
-	return folio_nr_pages((struct folio *)page);
-}
-
-/**
  * folio_next - Move to the next physical folio.
  * @folio: The folio we're currently operating on.
  *
_
